charts/spire/charts/spire-server/values.yaml

```diff
@@ -351,11 +347,59 @@ tornjak:
     successThreshold: 1
     timeoutSeconds: 5
 
+  # tornjak - Tornjak specific configuration
   config:
-    # -- persistent DB for storing Tornjak specific information
+    # -- Persistent DB for storing Tornjak specific information
     dataStore:
       driver: "sqlite3"
       file: "/run/spire/data/tornjak.sqlite3"
+
+    # Tornjak HTTP service
+    http:
+      # -- Enables Tornjak HTTP (insecure) service
+      enabled: true
+      # -- Container port value for HTTP
+      port: 10000
+      # -- Service to handle Tornjak HTTP connection
+      service:
+        type: ClusterIP
+        port: 10000
+        annotations: {}
+
+    # Tornjak TLS service
+    tls:
+      # -- Enables Tornjak TLS service
+      enabled: false
+      # -- Container port value for TLS
+      port: 20000
+      # -- Certificate for TLS connection
+      cert: "/opt/spire/sample-keys/cert.pem"
+      # -- Key for TLS connection
+      key:  "/opt/spire/sample-keys/key.pem"
+      # -- Service to handle Tornjak TLS connection
+      service:
+        type: ClusterIP
+        port: 20000
+        annotations: {}
+
+    # Tornjak mTLS service
+    mtls:
+      # -- Enables Tornjak TLS service
+      enabled: false
+      # -- Container port value for mTLS
+      port: 30000
+      # -- Certificate for mTLS connection
+      cert: "/opt/spire/sample-keys/cert.pem"
+      # -- Key for mTLS connection
+      key:  "/opt/spire/sample-keys/key.pem"
+      # -- CA for mTLS connection
+      ca: "/opt/spire/sample-keys/rootCA.pem"
```

Comment:
I'm assuming this is for validating the other side's certs? maybe userCA? Should consider just making this a name within a fixed location in the pod so the user doesn't have to know stuff such as /opt/spire/sample-keys?

Reply:
It should probably be in its own configmap/secret. Different systems do that part differently.

Reply:
This is used to build the ConfigMap `tornjak-config`, but you are right, we can hard code it there